src/components/MovieList.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import MovieCard from './MovieCard';
+import styles from './MovieList.module.css';
+import { Box, Select, MenuItem, FormControl, InputLabel } from '@mui/material';
+import Pagination from './Pagination';
+import FilterListIcon from '@mui/icons-material/FilterList';
+
+const MovieList = ({ searchResults }) => {
+  const [movies, setMovies] = useState([]);
+  const [page, setPage] = useState(1);
+  const [lastPage, setLastPage] = useState(1);
+  const [sort, setSortField] = useState('average_rating');
+  const [order, setSortOrder] = useState('desc');
+
+  useEffect(() => {
+    if (searchResults) {
+      setMovies(searchResults);
+    } else {
+      fetch(
+        `http://localhost:3001/movies?page=${page}&sort=${sort}&order=${order}`
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          setMovies(data.data);
+          setLastPage(data.lastPage);
+        })
+        .catch((error) => console.error('Error fetching movies:', error));
+    }
+  }, [page, sort, order, searchResults]);
+
+  const handleSortChange = (field, order) => {
+    setSortField(field);
+    setSortOrder(order);
+  };
+
+  return (
+    <Box className={styles.container}>
+      <Box className={styles.header}>
+        <h1>All Movies</h1>
+        <FormControl variant="outlined" className={styles.sortControl}>
+          <InputLabel>Sort by</InputLabel>
+          <Select
+            value={`${sort}-${order}`}
+            onChange={(event) => {
+              console.log(event.target.value);
+              handleSortChange(
+                event.target.value.split('-')[0],
+                event.target.value.split('-')[1]
+              );
+            }}
+            startAdornment={<FilterListIcon />}
+            label="Sort by"
+          >
+            <MenuItem value="release_date-asc">Release Date Ascending</MenuItem>
```

Comment:
Fixed. I used kebab case because in the backend it is how to access it, but now the kebab case is in the constants file and the consistency in the movielist.js is now snake case as i import the list_filter using the names i created as snake case